### PR TITLE
documentation: ZENKO-1763_Update_Minikube_versions

### PR DESCRIPTION
**Pull request by wabernat**
This PR updates all the version numbers required to run minikube (as requested).

It also fixes a lot of minor problems in the writing, and says more (I hope!) with less. 
This addresses #ZENKO-1763

### CODE REVIEW

File: docs/minikube.md [hunk 1]
```diff
@@ -1,75 +1,93 @@
-### Requirements
+# Deploying and Configuring Minikube
+
+## Requirements
 - [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
-- [minikube](https://github.com/kubernetes/minikube/#installation) v0.25
-  - [virtualbox](https://www.virtualbox.org/wiki/Downloads)
-- [helm](https://github.com/kubernetes/helm#install) v2.8.2
+- [minikube](https://github.com/kubernetes/minikube/#installation) v0.5
```
Comment (ssalaues) [inline docs/minikube.md]: Lets just do the latest and greatest. `v1.0.1` this is what I tested out on today.

File: docs/minikube.md [hunk 2]
```diff
@@ -1,75 +1,93 @@
...
+## Deploy Zenko
 
-Then you can run the following shell command to deploy a single node zenko stack with orbit enabled.
+Run the following shell command to deploy a single-node Zenko stack with Orbit enabled.
 ```shell
 $ helm install --name zenko \
   --set prometheus.rbac.create=false \
   --set zenko-queue.rbac.enabled=false \
   --set redis-ha.rbac.create=false \
   -f single-node-values.yml zenko
```
Comment (ssalaues) [inline docs/minikube.md]: The infamous `yml` vs `yaml`

**Review by ssalaues — COMMENTED** [review]
Minor stuff otherwise lgtm

File: docs/minikube.md (hunk 1 above)
Comment (wabernat) [reply docs/minikube.md]: done

File: docs/minikube.md (hunk 2 above)
Comment (wabernat) [reply docs/minikube.md]: done.

**Review by ssalaues — APPROVED** [review]

**Comment by wabernat** [comment]
@bert-e approve

**Comment by wabernat** [comment]
@bert-e reset

**Comment by wabernat** [comment]
@bert-e approve

**Review by philipyoo — APPROVED** [review]

**Review by dora-korpar — APPROVED** [review]

**Pull request merged**